sdk/include/reactos/subsys/sm/smmsg.h

```diff
@@ -1,17 +1,19 @@
 /*
- * PROJECT:         ReactOS Windows-Compatible Session Manager
- * LICENSE:         BSD 2-Clause License
- * FILE:            include/reactos/subsys/sm/smmsg.h
- * PURPOSE:         SMSS (SB and SM) Message Format
- * PROGRAMMERS:     Alex Ionescu
+ * PROJECT:     ReactOS NT-Compatible Session Manager
+ * LICENSE:     BSD 2-Clause License (https://spdx.org/licenses/BSD-2-Clause)
+ * PURPOSE:     SMSS Client (SB and SM) Message Format
```

Comment:
I know that SM stands for Session Manager, but what does SB stand for?

Grep doesn't seem to show anything: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=Session+B&sr=1

Reply:
That's **s**ession call**b**ack (session "back"), see the updated readme.txt . All callbacks made by SMSS to subsystems use conventionally this prefix.